ui/jobs/jobs.js

```diff
@@ -497,7 +497,8 @@ class Buff {
     if (initials.length == 2)
       txt = initials[0][0] + initials[1][0];
     else
-      txt = initials[0].slice(0, 3);
+      // These icon can hold only 2 CN/KR charactors. 3 will escape out of box
+      txt = initials[0].slice(0, 2);
```

Comment:
Is the code on line 496 is doing the right thing here for Chinese and Korean names? Or, is there some better way to shorten to two characters?

Reply:
line 496 `let initials = source.split(' ');` do not work for Chinese names.(I'm not very sure about Korean)
in JP/NA/EU, player's name is like `Suzune Tsukuyomi`, a space separates the last name from the first name.
But in CN, there is **no last name**, at most 6 charactors allowed in name, and space is not allowed.

There is a problem that CN player's name could also have English charactors. Look like this box can hold at most 4 English charactors/1 chinese charactor+2 English charactors.
Can it be like this: 
```
var pattern = new RegExp("[\u4E00-\u9FA5]+"); // A RegExp that matches Chinese charactors.
if (pattern.test(initials[0][0]) && pattern.test(initials[0][1]) && pattern.test(initials[0][2]))
  txt = initials[0].slice(0, 2); // If the first 3 charactor is both Chinese charactor.
else
  txt = initials[0].slice(0, 3);
```

Reply:
Instead of matching Chinese characters, it is better to filter English characters in Regex, I think.
```javascript
const pattern = /[\u21-\u7e]+/g;
if (pattern.test(initials[0].substr(0, 3))
   txt = initials[0].slice(0, 3);
else
  txt = initials[0].slice(0, 2);
```

May it be better?

Reply:
Korea haven't second part of name, too. just have 1 input line and it could be korean or english and NOT allowed name to short (like Una Chron --> U.C).
I'm not sure about Latin characters, but I've never seen before.